syntax = "proto3";

enum EntityType {
  T_Null = 0;
  T_Player = 1;
}

message EntityMove{
  EntityType t = 1;
  int32 entityId = 2;
  float x = 3;
  float y = 4;
  float z = 5;
}

//首次进入游戏
message PlayerFirstInGame{
  uint32 entityId = 1;
}

//权限
message GrantAuthorOfEntity{
  //  uint32 randomGrantId = 1;
  EntityMove entityMove = 2;
}
message AcceptGrant{
  //  uint32 randomGrantId = 1;
}